project/Newevaluation.html

```diff
@@ -5,8 +5,15 @@
   <meta charset="UTF-8">
   <link rel="stylesheet/less" type="text/css" href="assets/newevaluationStyle.less">
   <link rel="stylesheet/less" type="text/css" href="assets/common.less">
+
   <script src="assets/less.min.js" type="text/javascript"></script>
   <script src="src/views/common/common.js" type="text/javascript"></script>
+  <script src="src/data/technicalArea.js" type="text/javascript"></script>
```

Comment:
I am going to give you the same advice I gave Laura; This list has the potential to grow significantly and you will have to manage it. Try finding a way to group the imports based on different criteria so that it is easier to check if something is already imported or not.